tests/lib/collectors/gce_disk_copy.py

```diff
@@ -388,12 +388,12 @@ def testProcess(self,
     collector.PreProcess()
     conts = test_state.GetContainers(collector.GetThreadOnContainerType())
     for d in conts:
-      collector.Process(d)
+      collector.Process(d)  # pytype: disable=wrong-arg-types
```

Comment:
Can we add a note why we have to disable pytype here? Just because this has to be disabled throughout this file, be good to have a note about that for the first time you're disabling it for conts/d.

Reply:
Done.